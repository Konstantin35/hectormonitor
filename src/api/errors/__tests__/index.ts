import { InvalidAddressError, NoDataAddressError, ApiError } from '..';

describe('errors', () => {
  describe('InvalidAddressError', () => {
    it('throw default message', () => {
      const err = new InvalidAddressError();
      expect(err.message).toMatchSnapshot();
    });
  });

  describe('NoDataAddressError', () => {
    it('throw default message', () => {
      const err = new NoDataAddressError();
      expect(err.message).toMatchSnapshot();
    });
  });

  describe('ApiError', () => {
    it('throw default message', () => {
      const err = new ApiError();
      expect(err.message).toMatchSnapshot();
    });
  });
});
