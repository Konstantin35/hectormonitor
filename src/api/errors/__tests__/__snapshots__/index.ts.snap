// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`errors ApiError throw default message 1`] = `"Api returned an error"`;

exports[`errors InvalidAddressError throw default message 1`] = `"Invalid address"`;

exports[`errors NoDataAddressError throw default message 1`] = `"Address returns no data for the pool"`;
